scheduler/pkg/agent/client.go

```diff
@@ -48,6 +47,11 @@ import (
 	"google.golang.org/protobuf/encoding/protojson"
 )
 
+const (
+	// period for subservice ready "cron"
+	periodReadySubService = 10 * time.Second
```

Comment:
If each 2> subservice has 5 secs to be ready thats more than the 10 sec between ticks. Maybe doesn't matter?

Reply:
We could do the check in parallel perhaps? I can increase also the period of cron to 30seconds?